
# This is a message to hold data with position and trajectory information
Header header
float32 s
float32 ey
float32 epsi
float32 v
float32 x
float32 y
float32 v_x
float32 v_y
float32 psi
float32 psiDot
float32 a_x
float32 a_y
float32 u_a
float32 u_df
